en/guide/routing.md

```diff
@@ -68,19 +68,24 @@ app.all('/secret', (req, res, next) => {
 
 Route paths, in combination with a request method, define the endpoints at which requests can be made. Route paths can be strings, string patterns, or regular expressions.
 
-The characters `?`, `+`, `*`, and `()` are subsets of their regular expression counterparts. The hyphen (`-`) and the dot (`.`) are interpreted literally by string-based paths.
+{% capture caution-character %} In express 5, the characters `?`, `+`, `*`, `[]`, and `()` are handled differently than in version 4, please review the [migration guide](/{{ page.lang }}/guide/migrating-5.html#path-syntax) for more information.{% endcapture %}
 
-If you need to use the dollar character (`$`) in a path string, enclose it escaped within `([` and `])`. For example, the path string for requests at "`/data/$book`", would be "`/data/([\$])book`".
+{% include admonitions/caution.html content=caution-character %}
 
-<div class="doc-box doc-info" markdown="1">
+{% capture note-dollar-character %}In express 5, you don't need to escape the dollar character (`$`) with `\`.
```

Comment:
May be better worded as "In Express 4, regular expression characters such as `$` need to be escaped with a `\`."? Not strongly opinionated, but may be easier to explain it away as Express 4 everything could be a regex and Express 5 there are no regex supported (which is noted elsewhere already).

Reply:
I like your suggestion more